Traceback (most recent call last):
  File "f:\jinzhu\conda_env\luto\Lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "f:\jinzhu\conda_env\luto\Lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "f:\jinzhu\conda_env\luto\Lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "f:\jinzhu\conda_env\luto\Lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "f:\jinzhu\conda_env\luto\Lib\asyncio\base_events.py", line 687, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "f:\jinzhu\conda_env\luto\Lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "f:\jinzhu\conda_env\luto\Lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "f:\jinzhu\conda_env\luto\Lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# HIDE CODE


df = data.AGEC_CROPS.head(10)
df_styled = df.style.set_table_styles([
    {'selector': 'td', 'props': [('font-size', '10pt')]},
    {'selector': 'th', 'props': [('font-size', '10pt')]}
]).format(precision=2)

df_styled

------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mNameError[0m                                 Traceback (most recent call last)
Cell [1;32mIn[1], line 4[0m
[0;32m      1[0m [38;5;66;03m# HIDE CODE[39;00m
[1;32m----> 4[0m df [38;5;241m=[39m [43mdata[49m[38;5;241m.[39mAGEC_CROPS[38;5;241m.[39mhead([38;5;241m10[39m)
[0;32m      5[0m df_styled [38;5;241m=[39m df[38;5;241m.[39mstyle[38;5;241m.[39mset_table_styles([
[0;32m      6[0m     {[38;5;124m'[39m[38;5;124mselector[39m[38;5;124m'[39m: [38;5;124m'[39m[38;5;124mtd[39m[38;5;124m'[39m, [38;5;124m'[39m[38;5;124mprops[39m[38;5;124m'[39m: [([38;5;124m'[39m[38;5;124mfont-size[39m[38;5;124m'[39m, [38;5;124m'[39m[38;5;124m10pt[39m[38;5;124m'[39m)]},
[0;32m      7[0m     {[38;5;124m'[39m[38;5;124mselector[39m[38;5;124m'[39m: [38;5;124m'[39m[38;5;124mth[39m[38;5;124m'[39m, [38;5;124m'[39m[38;5;124mprops[39m[38;5;124m'[39m: [([38;5;124m'[39m[38;5;124mfont-size[39m[38;5;124m'[39m, [38;5;124m'[39m[38;5;124m10pt[39m[38;5;124m'[39m)]}
[0;32m      8[0m ])[38;5;241m.[39mformat(precision[38;5;241m=[39m[38;5;241m2[39m)
[0;32m     10[0m df_styled

[1;31mNameError[0m: name 'data' is not defined
NameError: name 'data' is not defined

